examples/04_glm_first_level/plot_hrf.py

```diff
@@ -1,65 +1,101 @@
-"""Example of hemodynamic response functions.
-==========================================
+"""
+Example of MRI response functions
+=================================
 
 Within this example we are going to plot the hemodynamic response function
-(hrf) model in SPM together with the hrf shape proposed by G.Glover, as well as
+(HRF) model in SPM together with the HRF shape proposed by G.Glover, as well as
 their time and dispersion derivatives.
+We also illustrate how users can input a custom response function,
+which can for instance be useful when dealing with non human primate data
+acquired using a contrast agent. In our case, we input a custom response function
+for MION, a common agent used to enhance contrast on MRI images of monkeys.
```

Comment:
Could you link to a reference explaining what is MION?

Reply:
I thought a link to MION may be to specific, so I added a link to the wikipedia page explaining what MRI contrast agents are!

Reply:
Sounds reasonable, thanks!